tests/requirements/test_hal_watchdog.py

```diff
@@ -77,11 +77,11 @@ def main():
 
     # Default watchdog timeout is 16s. Watchdog in this test is set to 3s, but timeout must be
     # few seconds higher to compensate for communication delays and MCU clock inaccuracies.
-    gdb.wait_for_reset(timeout=5)
+    gdb.wait_for_reset(timeout=10)
 
     finish_test(ssh)
 
 
 if __name__ == "__main__":
-    # logging.basicConfig(level=logging.INFO)
```

Comment:
Necessary?

Reply:
strange, i though it's uncommented
since we're not checking stdout but rather exit code, it's "good to have", but uncommented - for debugging failures